program/freedom-metal-program.wake

```diff
@@ -111,7 +135,8 @@ global target fixupDTS dts metalEntryPairOpt =
       | addChosenNode
       | addStdoutChosenFromAlias
       | (omap addEntry metalEntryPairOpt | getOrElse (_))
-      | addHfclk clkFrequency
+      | (omap "rom".addMetalMemory romNode | getOrElse (_))
```

Comment:
did you mean to take out the `addHfclk` call?

Reply:
good catch. didn't mean to do that in this PR